tests/Tests/500__jupyterhub/test.robot

```diff
@@ -75,7 +75,28 @@ Can Launch Python3
    [Tags]  Sanity  TBC
    Launch Python3 JupyterHub
 
+Verify Message That Image Builds Are In Progress
+    [Documentation]     Verifies that Image Builds In Progress are Shown In RHODS Dashboard
+    [Tags]      Sanity
+    ...         Tier2
+    ...         ODS-460
+    ${build_name}=  Search Last Build  namespace=redhat-ods-applications    build_name_includes=pytorch
+    Delete Build    namespace=redhat-ods-applications    build_name=${build_name}
+    ${new_buildname}=  Start New Build    namespace=redhat-ods-applications    buildconfig=s2i-pytorch-gpu-cuda-11.4.2-notebook
+    Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${new_buildname}   expected_status=Running
+    Launch Dashboard   ocp_user_name=${TEST_USER.USERNAME}    ocp_user_pw=${TEST_USER.PASSWORD}   ocp_user_auth_type=${TEST_USER.AUTH_TYPE}   dashboard_url=${ODH_DASHBOARD_URL}   browser=${BROWSER.NAME}   browser_options=${BROWSER.OPTIONS}
+    Verify Given Message In RHODS Dashboard Notifications  message=Notebook images are building
+    Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${new_buildname}   expected_status=Complete
```

Comment:
A suggestion for making this TC more clean/readable:
1. create `Delete Last Pytorch Build` keyword to contain line 83-84
2. create `Start New Pytroch Build` kw to contain line 85-86
3. create `Wait Until Pytorch build is completed` to contain line 89 
4. leave the remaining ones.

Reply:
Is it necessary to create a new keyword for Line 89 ? Because I am using an already written keyword.

Reply:
it's not required, you can leave it as it is

Reply:
Updated